content/en/contribute/code/style-guide.md

```diff
@@ -484,4 +484,34 @@ providing a length property for a collection class.
 
 [sideeffect]: http://en.wikipedia.org/wiki/Side_effect_(computer_science)
 
+## Github Actions
+
+### Managing Secrets
+
+Secrets are encrypted for use for things like passwords or API keys. They can be added at the org, repo, or repository level. They must be added by an admin. 
```

Comment:
Duplication of "repo" and "repository"...

```suggestion
Secrets are encrypted for use for things like passwords or API keys and can be added at the org or repository level. They must be added by an admin. 
```

Reply:
thanks I was pulling from this statement and didn't notice it. Now I need to know what the difference is. 

> Encrypted secrets allow you to store sensitive information in your organization, repository, or repository environments.

Reply:
Repo environments means one repo could have multiple environments it deploys to, for example, in CD you could have a testing environment that gets pushed nightly, and then a prod environment that gets pushed on tag. We haven't used that feature yet.